src/Styleguide/Pages/Artist/Routes/RelatedArtists/RelatedArtistsRefetchContainer.tsx

```diff
@@ -110,13 +110,7 @@ export const RelatedArtistsRefetchContainer = createRefetchContainer(
                         ({ node }) => {
                           return (
                             <Box p={1} width={width}>
-                              <ArtistCard
-                                src={node.image.cropped.url}
-                                headline={node.name}
-                                subHeadline={
-                                  node.formatted_nationality_and_birthday
-                                }
-                              />
+                              <ArtistCardFragmentContainer {...node as any} />
```

Comment:
Do this instead, I think?

```tsx
<ArtistCardFragmentContainer artist={node} />
```

Reply:
Ah, indeed that works without `@relay(mask: false)` (and updating the props as well).